googlefonts/roboto-classic
# Build Roboto Condensed VF

Currently, the build will only output the Roboto family. This PR will also build the Roboto Condensed VF family.

I've also updated the following:
- Update dependencies to latest fontmake, fonttools etc
- Add SemiBold STAT AxisValue
- Update tests to use latest fontbakery

We want to push Roboto Condensed VF to Google Fonts before Roboto VF because we can see if we've missed anything on a much smaller audience.

I still need to version bump the fonts.
